CHANGELOG.md

```diff
@@ -2,6 +2,112 @@
+- structlog: Deprecates the `ddtrace.contrib.structlog` module. Use attributes exposed in `ddtrace.contrib.structlog.__all__` instead.
+- subprocess: Deprecates the `ddtrace.contrib.subprocess` module. Use attributes exposed in `ddtrace.contrib.subprocess.__all__` instead.
+- tornado: Deprecates all modules in the `ddtrace.contrib.tornado` package. Use attributes exposed in `ddtrace.contrib.tornado.__all__` instead.
+- urllib: Deprecates the `ddtrace.contrib.urllib` module. Use attributes exposed in `ddtrace.contrib.urllib.__all__` instead.
+- urllib3: Deprecates the `ddtrace.contrib.urllib3` module. Use attributes exposed in `ddtrace.contrib.urllib3.__all__` instead.
+- vertica: Deprecates the `ddtrace.contrib.vertica` module. Use attributes exposed in `ddtrace.contrib.vertica.__all__` instead.
+- webbrowser: Deprecates the `ddtrace.contrib.webbrowser` module. Use attributes exposed in `ddtrace.contrib.webbrowser.__all__` instead.
+- wsgi: Deprecates the `ddtrace.contrib.wsgi` module. Use attributes exposed in `ddtrace.contrib.wsgi.__all__` instead.
+- yaaredis: Deprecates all modules in the `ddtrace.contrib.yaaredis` package. Use attributes exposed in `ddtrace.contrib.yaaredis.__all__` instead.
+### Bug Fixes
+
+- CI Visibility: This fix resolves an issue where exceptions other than timeouts and connection errors raised while fetching the list of skippable tests for ITR were not being handled correctly and caused the tracer to crash.
+- CI Visibility: Fixes a bug where `.git` was incorrectly being stripped from repository URLs when extracting service names, resulting in `g`, `i`, or `t` being removed (eg: `test-environment.git` incorrectly becoming `test-environmen`)
+- LLM Observability: This fix resolves an issue where custom trace filters were being overwritten in forked processes.
+- Fix for a side-effect issue with module import callbacks that could cause a runtime exception.
+- llmobs: This fix resolves an issue where session_id was being defaulted to trace_id which was causing unexpected UI behavior.- LLM Observability: This fix resolves an issue where LLM Observability spans were not being submitted in forked processes, such as when using `celery` or `gunicorn` workers. The LLM Observability writer thread now automatically restarts when a forked process is detected.
+- Fixed an issue with some module imports with native specs that don't support attribute assignments, resulting in a `TypeError` exception at runtime.
+- This fix resolves an issue where `ddtrace` package files were published with incorrect file attributes.
+- tracer: This fix resolves an issue where django db instrumentation could fail.
+- openai: Fixes a bug where \`asyncio.TimeoutError\`s were not being propagated correctly from canceled OpenAI API requests.
+
+- tracing: Adds `ddtrace.trace.Context` to the public api. This class can now be used to propagate context across execution boundaries (ex: threads).
```

Comment:
Should this be a new feature? I see why it's marked as a fix, but it is a public-facing change... 🤔 Thoughts @mabdinur ?

Reply:
Feature!